smac/optimizer/smbo.py

```diff
@@ -196,20 +196,13 @@ def run(self):
                            configuration_space=self.config_space,
                            logger=self.logger)
 
-            start_time = time.time()
-            X, Y = self.rh2EPM.transform(self.runhistory)
-
-            self.logger.debug("Search for next configuration")
-            # get all found configurations sorted according to acq
-            challengers = self.choose_next(X, Y)
-
-            time_spent = time.time() - start_time
-            time_left = self._get_timebound_for_intensification(time_spent)
+            time_left = self._get_timebound_for_intensification()
```

Comment:
Hm, I'm not sure if this will still work now that doesn't take the time for `choose_next` into account any more.